www/docs/customization/index.md

```diff
@@ -58,5 +58,14 @@ for `__VERSION__` (latest):
     https://raw.githubusercontent.com/goreleaser/goreleaser/__VERSION__/www/docs/static/schema-pro.json
     ```
 
+[^goreleaser-yaml]:
+    While most of the documentation refers to the `.goreleaser.yaml` filename
+    for simplicity, a few different variants of it are actually accepted.
+    In order of precedence:
+    * `.config/goreleaser.yml`
+    * `.config/goreleaser.yaml`
+    * `.goreleaser.yml`
+    * `.goreleaser.yaml`
```

Comment:
```suggestion
    * `.config/goreleaser.yml`
    * `.config/goreleaser.yaml`
    * `.goreleaser.yml`
    * `.goreleaser.yaml`
    * `goreleaser.yml`
    * `goreleaser.yaml`
```

It also looks for `goreleaser.ya?ml` (without the preceding `.`).
See https://github.com/goreleaser/goreleaser/blob/59dea954bcce6d7c6868ad153bd11f9e4faac95f/cmd/config.go#L46-L53